app/src/main/java/org/fossasia/openevent/app/common/contract/Pipe.java

```diff
@@ -0,0 +1,7 @@
+package org.fossasia.openevent.app.common.contract;
+
+public interface Pipe<T> {
```

Comment:
To push the ticket from viewholder to adapter. It is essentially a Java 8 or Rx consumer, but Java 8 one does not work on Android and Rx one needs to throw exception.